03/app.js

```diff
@@ -11,4 +11,21 @@ const buttonSettings = {
         color: '#444'
     },
     text: 'Click me!',
-}
\ No newline at end of file
+}
+
+const button = document.createElement('button');
+const htmlHook = document.getElementsByClassName('parent-for-button');
+
+for(const props in buttonSettings){
+    if(typeof buttonSettings[props] === 'object'){
+        if(buttonSettings[props].className !== undefined) button.classList.add(buttonSettings[props].className);
+        if(buttonSettings[props].title !== undefined) button.setAttribute('title', buttonSettings[props].title);
+        if(buttonSettings[props].border !== undefined) button.style.border = buttonSettings[props].border;
+        if(buttonSettings[props].padding !== undefined) button.style.padding = buttonSettings[props].padding;
+        if(buttonSettings[props].color !== undefined) button.style.color = buttonSettings[props].color;
```

Comment:
Pisanie takiej ilości `if`-ów nie jest dobrym pomysłem. Przy 100 elementach byłoby to bardzo uciązliwe.
Lepszym rozwiązaniem byłoby zrobienie pętli tylko dla elementów, które możemy wygodnie wykorzystać np. `for( const prop in buttonSettings.css)`